lib/util/http/github.ts

```diff
@@ -154,11 +154,6 @@ function handleGotError(
   ) {
     return err;
   }
-  if (err.statusCode === 404) {
-    logger.debug({ url: path }, 'GitHub 404');
-  } else {
-    logger.debug({ err }, 'Unknown GitHub error');
-  }
```

Comment:
These will be logged lower down (debug in http layer) and higher up (lookup error)